SPEC.md

```diff
@@ -115,34 +118,62 @@ The dependencies are applied in order and each image dependency can overwrite fi
 Execution details specified in image dependencies are ignored.
 An optional *path whitelist* can be provided, in which case all non-specified files from all dependencies will be omitted in the final, assembled rootfs.
 
-Image Format TODO
+## App Container Executor
```

Comment:
*waves hands* This seems better to me as coming after the Pods section. Additionally, the section that is titled "Executor Perspective" should just follow on after the introduction of this section (i.e. also remove the "Executor Perspective" subsection)